app/services/usage-statistics.ts

```diff
@@ -129,7 +142,7 @@ export class UsageStatisticsService extends Service {
       product: 'SLOBS',
       version: this.version,
       count: 1,
-      uuid: this.userService.state.auth ? this.userService.state.auth.platform.id : void 0,
+      uuid: this.userService.state.auth ? this.userService.state.auth.platform.id : this.state.uuid,
```

Comment:
What's the purpose of this?  Can we just send `slobs_user_id` in this field instead?  It's literally just a persistent uuid, which seems to be what this is.

Reply:
It's for combining analytics events from unauthorized users who don't have `slobs_user_id` .
I'll add a comment here